### PR TITLE
update contributing instructions

### PR DESCRIPTION
**Pull request by fricklerhandwerk**
running devmode with `--run` doesn't actually work, it doesn't pick up
the changes

Made with @roberth at #ZurichZHF

### CODE REVIEW

**Review by infinisil — APPROVED** [review]
Looking good, except that devmode takes minute to reload instead of just a few seconds :sweat_smile:

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,6 +8,26 @@ We strongly encourage everyone interested to participate:
...
+and open a browser at <http://localhost:5500>.
+
+As you make changes, your browser should auto-reload within a few seconds.
+
+To manually test [redirects](./_redirects):
+
+```shell-session
+[nix-shell:nix.dev]$ nix-build -A build
+[nix-shell:nix.dev]$ cd result
+[nix-shell:nix.dev]$ netlify dev
```
Comment (infinisil) [inline CONTRIBUTING.md:28]: Actually, that doesn't work, `cd`ing into `result` unloads direnv!

Comment (fricklerhandwerk) [reply CONTRIBUTING.md:28]:
```suggestion
[nix-shell:nix.dev]$ netlify dev -d result
```

Cool, thanks a lot!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -8,6 +8,25 @@ We strongly encourage everyone interested to participate:
...
+
+Enter the development environment with `nix-shell`, or [set up direnv](https://nix.dev/guides/recipes/direnv.html)` and run `direnv allow`, and then:
+
+```shell-session
+[nix-shell:nix.dev]$ devmode
+```
+
+and open a browser at <http://localhost:5500>.
+
+As you make changes, your browser should auto-reload within a few seconds.
```
Comment (fricklerhandwerk) [inline CONTRIBUTING.md:21]:
```suggestion
As you make changes, your browser should auto-reload.
```

**Pull request merged**